ci/docker/os-image-stemcell-builder/scripts/update.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 set -ex
-
+if [ `uname -m` != "ppc64le" ]; then
```

Comment:
Sorry, I will remove it from this PR, it is part of another possible PR related to CI. Answering to your question, for ppc64le it should be
```
deb http://ports.ubuntu.com/ubuntu-ports/ trusty main restricted
deb-src http://ports.ubuntu.com/ubuntu-ports/ trusty main restricted
...
deb http://ports.ubuntu.com/ubuntu-ports/ trusty-security universe
deb-src http://ports.ubuntu.com/ubuntu-ports/ trusty-security universe
``` 
and have repos already in base ppc64le/ubuntu:14.04 image